library/src/main/java/com/leinardi/android/speeddial/SpeedDialView.java

```diff
@@ -757,10 +770,35 @@ private void enlargeAnim(View view, long startOffset) {
         view.startAnimation(anim);
     }
 
+    /**
+     * SpeedDial closing animation.
+     *
+     * @param view view that starts that animation.
+     */
+    private void shrinkAnim(final View view, long startOffset) {
```

Comment:
Please move this in the `UiUtil` class, together with the other `shrinkAnim`. I have to fine a better naming for these methods, but for now just move them together.

- [ ] move in  `UiUtil`

Reply:
@leinardi If I move the `shrinkAnim` to `UiUtil` class, should I move the `enlargeAnim` too?